environment-api/src/main/java/com/sequenceiq/environment/api/doc/environment/EnvironmentModelDescription.java

```diff
@@ -102,6 +102,9 @@
     public static final String RESOURCE_GROUP_USAGE = "Resource group usage: single resource group for all resources where possible "
             + "or use multiple resource groups.";
 
+    public static final String KEY_VAULT_ID = "Id of the Azure key vault which contains CustomerManagedkey to encrypt resources";
+    public static final String KEY_NAME = "Name of the CustomerManagedkey to encrypt Azure resources";
```

Comment:
Is it really the key name that is required?! I though we needed the exact key version URL (see the associated [REST API](https://docs.microsoft.com/en-us/rest/api/compute/diskencryptionsets/createorupdate)).

Reply:
Agree, Shall I consider changing the params keyVaultId and keyName to 'keyVaultName' and 'KeyURI'  respectively?

Reply:
keyvaultUrl?

Reply:
As mentioned in my other comment below, we need the Resource ID of the source KeyVault, so `KEY_VAULT_ID` sounds alright (but its semantics & validation needs to be changed).